Aspen-Discovery/aspen-discovery
# DIS-162 Update Boundless/Axis360 get availability API URL and response

Updated the Boundless/Axis360 gets records availability API URL because the old one has been deprecated, and it caused the Boundless record on Aspen to sync incorrectly with the library's boundless catalog. By updating the API URL and adapting the response processing, this PR makes sure to get the correct records' availability.

Test note:
1. Before PR, search boundless records in the Aspen catalog and compare to the same record availability in the boundless catalog, notice that the availability statuses are different
2. Apply PR, and run full update on boundless in the Aspen boundless settings
3. Search boundless records in the Catalog and compare to the same record availability in the boundless catalog, the availability is the same.
